src/MSALWrapper/AuthFlow/Web.cs

```diff
@@ -80,7 +80,6 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
                 this.errors.Add(ex);
                 this.logger.LogDebug($"Initial ${this.Name()} auth failed. Trying again with claims.\n{ex.Message}");
 
-

```

Comment:
This change can be reverted. It's seem line removed by mistake.

Reply:
I will address that in `[9/🧹]` - a final linting PR to catch up on all the small linting warnings.